src/ape/api/providers.py

```diff
@@ -145,6 +182,12 @@ def _set_description(self):
 
 @abstractdataclass
 class ReceiptAPI:
+    """
+    An Abstract class to represent ReceiptAPI.
+    Receipt is used to find a new contract Address and its many attributes.
```

Comment:
I don't understand this part, I don't think that is true?

Reply:
[Take a look at the last two properties. A simple use of a receipt is to find out a new contract's contractAddress. A more advanced used for a receipt is with Proving the Existence of Logs to the Blockchain](https://ethereum.stackexchange.com/questions/16525/what-are-ethereum-transaction-receipts-and-what-are-they-used-for/16541)